### PR TITLE
Adding the abillity to define the field width. 

### PR DESCRIPTION
**Pull request by avifatal**
Some time you want fields to be in the same row.
So now I made it possible via

`$form->setContainerWidth(int);` and `$field->setContainerWidth(int);`

The reason I did not use the existing `setWidth` function is because it is not backward compatibile.
adding a third variable will force you to push `null,null` to each field you want to  set a container width.

here is an example...

image

### CODE REVIEW

File: views/form.blade.php [hunk 1]
```diff
@@ -14,9 +14,12 @@
             @if(!$tabObj->isEmpty())
                 @include('admin::form.tab', compact('tabObj'))
             @else
-                <div class="fields-group">
+                <div class="fields-group row" style=" display: flex;flex-wrap: wrap;">
```
Comment (Ladel) [inline views/form.blade.php]: `style=" display: flex;flex-wrap: wrap;"` don't need

File: src/Form/Builder.php [hunk 2]
```diff
@@ -427,8 +452,8 @@ public function submitButton()
         $text = trans('admin::lang.submit');
 
         return <<<EOT
-<div class="btn-group pull-right">
-    <button type="submit" class="btn btn-info pull-right" data-loading-text="<i class='fa fa-spinner fa-spin '></i> $text">$text</button>
```
Comment (Ladel) [inline src/Form/Builder.php]: The old look better

File: views/form.blade.php [hunk 3]
```diff
@@ -14,9 +14,12 @@
             @if(!$tabObj->isEmpty())
                 @include('admin::form.tab', compact('tabObj'))
```
Comment (Ladel) [inline views/form.blade.php]: It should be added in the tab

File: src/Form/Field.php [hunk 4]
```diff
@@ -350,6 +352,23 @@ public function setWidth($field = 8, $label = 2)
     }
 
     /**
+     *
+     * @param int $width
+     *
+     * @return $this
+     */
+    public function setContainerWidth($width){
+        $this->containerWidth = $width;
```
Comment (Ladel) [inline src/Form/Field.php]: return $this;

File: views/form.blade.php (hunk 3 above)
Comment (avifatal) [inline views/form.blade.php]: I disagree, it is suppose to be simple, horizontal is a basic thing. tabs for fields are over complicated.
BTW, sleeping owl has the worst API for horizontal fields

Comment (Ladel) [inline views/form.blade.php]: http://prntscr.com/euj9gy 
It's work, simple.
[Demo horizontal fields](https://github.com/SleepingOwlAdmin/demo/blob/master/admin/Http/Sections/Contacts5.php#L116)

File: views/form.blade.php (hunk 1 above)
Comment (avifatal) [inline views/form.blade.php]: You need it when you don't have rows for each 12 cols. 
http://stackoverflow.com/a/42607629/936651

File: src/Form/Builder.php (hunk 2 above)
Comment (avifatal) [inline src/Form/Builder.php]: it is because the form can be stretched.

File: views/form.blade.php (hunk 3 above)
Comment (avifatal) [inline views/form.blade.php]: Very bad API, I made few application with sleeping owl... nightmare.
I can implement this, but I think its over engineering 
This is why i'm here...

File: src/Form/Field.php (hunk 4 above)
Comment (avifatal) [inline src/Form/Field.php]: Fixed.

**Comment by avifatal** [comment]
@z-song , can this be merged?

File: views/form.blade.php (hunk 3 above)
Comment (isometriq) [inline views/form.blade.php]: I agree for tabs, mainly for consistency

**Pull request closed without merging**